purchase_requisition_from_rfq/models/purchase_requisition.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class PurchaseRequisition(models.Model):
+    _inherit = "purchase.requisition"
+
+    purchase_ids = fields.One2many(
+        default=lambda self: self._context.get('active_ids', []),
+    )
+    line_ids = fields.One2many(
+        default=lambda self: self._get_line(),
+    )
+    from_rfq = fields.Boolean(
+        default=False,
+        readonly=True,
```

Comment:
help="True, if this purchase agreement is created from RFQs",

Reply:
Added